Assets/Plugins/LeapMotion/Core/Editor/LeapServiceProviderEditor.cs

```diff
@@ -61,79 +80,205 @@ public class LeapServiceProviderEditor : CustomEditorBase<LeapServiceProvider> {
     }
 
     public override void OnInspectorGUI() {
-      #if UNITY_2019_3_OR_NEWER
+
+#if UNITY_2019_3_OR_NEWER
       // Easily tracking VR-enabled-or-not requires an XR package installed, so remove this warning for now.
-      #else
+#else
       if (UnityEditor.PlayerSettings.virtualRealitySupported && !isVRProvider) {
         EditorGUILayout.HelpBox(
           "VR support is enabled. If your Leap is mounted to your headset, you should be "
           + "using LeapXRServiceProvider instead of LeapServiceProvider. (If your Leap "
           + "is not mounted to your headset, you can safely ignore this warning.)",
           MessageType.Warning);
       }
-      #endif
+#endif
 
       base.OnInspectorGUI();
     }
 
     public virtual void OnSceneGUI() {
-      Vector3 origin = target.transform.TransformPoint(controllerOffset);
 
-      Vector3 local_top_left, top_left, local_top_right, top_right,
-              local_bottom_left, bottom_left, local_bottom_right, bottom_right;
-      getLocalGlobalPoint(-1, 1, 1, out local_top_left, out top_left);
-      getLocalGlobalPoint(1, 1, 1, out local_top_right, out top_right);
-      getLocalGlobalPoint(-1, 1, -1, out local_bottom_left, out bottom_left);
-      getLocalGlobalPoint(1, 1, -1, out local_bottom_right, out bottom_right);
+      switch (GetSelectedInteractionVolume()) {
+        case LeapServiceProvider.InteractionVolumeVisualization.None:
+          break;
+        case LeapServiceProvider.InteractionVolumeVisualization.LeapMotionController:
+          DrawLeapMotionControllerInteractionZone(LMC_BOX_WIDTH, LMC_BOX_DEPTH, LMC_BOX_RADIUS, Color.white);
+          break;
+        case LeapServiceProvider.InteractionVolumeVisualization.Rigel:
+          DrawRigelInteractionZoneMesh();
+          break;
+        case LeapServiceProvider.InteractionVolumeVisualization.Automatic:
+          DetectConnectedDevice();
+          break;
+        default:
+          break;
+      }
+
+    }
+
+    private void ParseRigelInteractionMeshData() {
+
+      if (_rigelInteractionZoneMesh == null) {
+        _rigelInteractionZoneMesh = (Mesh)AssetDatabase.LoadAssetAtPath(Path.Combine("Assets", "UnityModules", "Assets", "Plugins", "LeapMotion", "Core", "Models", "Rigel-interaction-cone-placeholder.obj"), typeof(Mesh));
```

Comment:
There is an extra `"Assets", "UnityModules",` prepended to this Path (and thus the Editor is not able to find it)...